packages/cinterop/build.gradle.kts

```diff
@@ -113,11 +113,7 @@ val nativeLibraryIncludesIosSimulatorArm64Release =
     includeBinaries(releaseLibs.map { "$absoluteCorePath/build-simulator-arm64/lib/$it" })
 
 kotlin {
-    jvm {
-        compilations.all {
-            kotlinOptions.jvmTarget = Versions.jvmTarget
-        }
-    }
+    jvm()
```

Comment:
Is it getting the correct target default?

Reply:
Yes, it is being set by https://github.com/realm/realm-kotlin/pull/1514/files#diff-5ea5ab9e6b701c3d35d3fd2fad81ff600102db520494994cc7448e7d3e032d9dR32 ... which now controls everything.